### ISSUE TITLE
Default import

### ISSUE THREAD

**Issue by BenBrostoff** [TheBrainFamily/wait-for-expect#8]
First off, great library and thanks for this.

I noticed when requiring this package I get an object that looks like below:

```
> const waitFor = require('wait-for-expect');
> waitFor
{ default: [Function: waitForExpect] }
```

This isn't a bug because I can just destructure:

```
const { default: waitFor } = require('wait-for-expect');
```

but it might help just for convenience purposes to change how the function is exposed. I'm using this for a server-side project without anything babel related, so this probably works fine for a certain use that isn't mine right now :-).

**Comment by lgandecki** [comment]
Hmmm... I think this is typescript somehow messing it up... if you would like to research this a bit and provide a PR that would be fantastic :)

**Comment by lgandecki** [comment]
closed in #9
